src/frontend/src/lib/components/ui/DropdownButton.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import { IconExpandMore } from '@dfinity/gix-components';
+
+	export let disabled = false;
+	export let button: HTMLButtonElement | undefined = undefined;
+	export let ariaLabel: string;
+	export let opened = false;
```

Comment:
Instead of `opened` what do you think of the name `focused`?

Reply:
the idea was to really be the indicator that the dropdown is open/visible, in fact I am using it to flip the icon too

in the sense, it could be focused but not opened. Maybe a better name is `dropdownOpen`? or `clicked`? boh

Reply:
No no it's fine, `opened` works, thanks.